### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by Marrakchino** [pietermartin/umlg#47]
The github repository (and the website as well) is very poorly-documented. Maybe add a more complete README file.

**Comment by pietermartin** [comment]
Yeah, the documentation for Umlg is totally unmaintained at present.
Between the day job and Sqlg I simply gave up on that one.
However I do work fairly often on the code base as its in production use at the said day job.

Can't say when I'll get to Umlg docs but I still have many future plans for UMLG. Basically as Sqlg matures many things become possible for Umlg.
